bedrock/firefox/firefox_details.py

```diff
@@ -157,28 +167,25 @@ def get_download_url(self, platform, language, version, product='firefox'):
             language = 'ja-JP-mac'
 
         if version == self.latest_version('aurora'):
-            return self._get_aurora_download_url(platform, language, version)
+            # Download links are different for localized versions
+            if language == 'en-US':
+                if platform == 'Windows':
```

Comment:
Should this now be `if platform.startswith('Windows'):`? The 64bit build should probably also use the stub right?